app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def last_5_comments
```

Comment:
- Good Job 👍 
- 
![Capture](https://user-images.githubusercontent.com/63972670/199861465-da57f535-dbe2-423d-bff3-ad01dde2af34.JPG)

-

Reply:
Thank you so much for your review @OLIPLICHE !

If you want to give it a try, run my seems.rb 
I used a gen called Faker, it's pretty cool!